Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/PowerSupplies/DieselEngine.cs

```diff
@@ -1085,7 +1085,8 @@ public void Update(float elapsedClockSeconds)
             if ((State != DieselEngineState.Starting) && (RealRPM == 0f))
                 State = DieselEngineState.Stopped;
 
-            if ((State == DieselEngineState.Stopped) || (State == DieselEngineState.Stopping) || ((State == DieselEngineState.Starting) && (RealRPM < StartingRPM)))
+            // fuel consumption will occur when engine is running above the starting rpm
+            if ((State == DieselEngineState.Stopped) || ((State == DieselEngineState.Stopping) && (RealRPM < StartingRPM)) || ((State == DieselEngineState.Starting) && (RealRPM < StartingRPM)))
```

Comment:
&& takes precedence over ||, and == takes precedence over both of these, so brackets are totally unnecessary in this line except the if()'s brackets. See: [Operator precedence](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/#operator-precedence)

Reply:
Whilst I agree that it is possible to remove the brackets, sometimes I prefer a long hand approach as I believe that it makes the code more readable for all.

So I have removed some of the brackets and left a couple on this basis.

Thanks for highlighting this.

Could you please approve this PR now.